app/assets/javascripts/autocise_textarea.js

```diff
@@ -0,0 +1,53 @@
+// Get the max height and apply
+// to all textareas when page loads/reloads
+
+$(document).ready(function() {
+  var numberArray = [];
+  $('textarea').each(function(textarea) {
```

Comment:
Can we use a class name on this? Currently this will happen on every textarea on the site, even on pages where this might not be necessary in the future. I recommend adding a new class shared between all the textareas you want to have with this behavior.

You'll probably also want to remove the textarea in the parentheses for your anonymous function. It's an unused variable and might conflict with other textareas that are named in this space.

Reply:
for some reason would not except a class name in this instance (believe me I tried)

Reply:
What are the other textareas on the site?

Reply:
can you use `$('.vocab_sheet textarea')` to find it? This selector is used in vocab_sheet.js